sw/cheri/common/platform-pinmux.hh

```diff
@@ -300,6 +302,7 @@ class SonataPinmux : private utils::NoCopyNoMove {
   static constexpr uint8_t block_input_options(BlockInput block_input) {
     switch (block_input) {
       case BlockInput::SpiReceive3:
+        return 4;
```

Comment:
Why is this `4` but the one below is `3`? Both seem to have 4 CS lines wire up in [top_sonata](https://github.com/lowRISC/sonata-system/blob/9a87f724f376adb3400cbe492052ba69eefc3663/rtl/fpga/top_sonata.sv#L272)